tests/unit_tests/test_vote.py

```diff
@@ -79,7 +79,7 @@ def fixture_votes_fpath(votes, tmp_path_factory):
 
 def test_add_ranks(ncbi, votes, tmp_path):
     out = tmp_path / "taxonomy.ranks_added.tsv"
-    df = vote.add_ranks(df=votes, out=out, ncbi=ncbi)
+    df = vote.add_ranks(df=votes, ncbi=ncbi)
```

Comment:
Include a copyright notice and script explanation at the top.

Reply:
Same comments as above